dev-tools/travis/print-errors-from-test-reports.py

```diff
@@ -55,6 +55,10 @@ def print_error_reports_from_report_file(file_path):
         if fail is not None:
             print_detail_information(testcase, fail)
 
+        failure = testcase.find("failure")
```

Comment:
I would suspect the validity of this change since "fail" is a substring of "failure", the script should not miss "failure"s by searching "fail".